geniza/corpus/forms.py

```diff
@@ -200,6 +200,10 @@ class DocumentSearchForm(forms.Form):
         # Translators: label for document type search form filter
         label=_("Document Type"),
     )
+    has_image = BooleanFacetField(
+        # Translators: label for "has transcription" search form filter
```

Comment:
looks like you copied the comment and forgot to update it :-) 

```suggestion
        # Translators: label for "has image" search form filter
```